core/lib/jobs/jobs/jobs.py

```diff
@@ -219,28 +219,66 @@ def _get_placeholders_str(num_placeholders, start=1):
     return ", ".join(f"${i}" for i in range(start, start + num_placeholders))
 
 
+async def get_customer_quota(con, customer_id, service) -> Dict[str, int]:
+    """Query DB and return usage limit and current usage.
+    Returns:
+        - Dictionary with account limits and account usage
+    """
+    # get service specific usage restrictions for the customer account
+    # uploads limit, jobs limit, end date of plan
+    usage_limit_query = "SELECT usage_restrictions->$1 AS usage FROM customers WHERE id=$2"
```

Comment:
There is a lot of unnecessary data getting queried here. For example, `total_jobs` from `current_usage_query` isn't used at all. There's also 3 queries used to calculate the current usage, but that can be achieved with just one query:
```
SELECT COUNT(*) AS total_uploads, SUM(jobs_count) AS total_jobs
FROM (
    SELECT (
        CASE WHEN (COUNT(*) <= 2 AND COUNT(*) > 0) THEN 1 ELSE COUNT(*) END
    ) AS jobs_count FROM jobs_result
    WHERE customer_id=$1 AND type=$2
    GROUP BY upload_id
) dt;
```